docs/source/releases/v1_11_7a0.rst

```diff
@@ -19,6 +19,7 @@ Version 1.11.7a0 (2023-10-11)
 * Add sphinx reference links throughout geoIPS documentation
 * Rename Geostationary Sectors
 * Create "output_checkers" interface
+* Create "output_checker_kwargs" commandline argument
```

Comment:
Adding a dummy conversation so we don't accidentally merge this before #368 is merged.